src/lib/utils/envHelper.ts

```diff
@@ -0,0 +1,96 @@
+export interface IENVS {
+    NETWORK: string,
+    BEACONCHA_URL: string,
+    LAUNCHPAD_URL: string,
+    ETHERSCAN_URL: string,
+}
+
+export type NetworkDataType = {
+    logo: string;
+    networkId: number;
+    optionLabel: string;
+    activeLabel: string;
+    ssvApiEndpoint: string;
+    ssvContractAddress: string;
+    setterContractAddress: string;
+    getterContractAddress: string;
+};
+
+export const NETWORKS = {
+  MAINNET: 1,
+  GOERLI: 5,
+};
+
+const NETWORK_VARIABLES =  {
+        [NETWORKS.MAINNET]: {
+            logo: 'dark',
+            activeLabel: 'Ethereum',
+            optionLabel: 'Ethereum Mainnet',
+    },
```

Comment:
On production will be two testnets and one mainnet, keys should be based on `<version>_<network>` pair

Reply:
done